Automattic/vip-go-mu-plugins
# Set default Jetpack version to use the 9.1 subtree

## Description

Set the default Jetpack version to 9.1

NOTE: This is blocked by #1843, and should only be merged when it's the agreed release time.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR (once #1843 has been merged)
1. Check that the Jetpack version in the dashboard is 9.1 (not 9.1-beta, not 9.0.2.1)
1. Check that everything keeps working
